.github/workflows/pr-commands-closed.yml

```diff
@@ -15,4 +17,4 @@ jobs:
         echo PR #${{ github.event.number }} has been closed without being merged, removing milestone.
         gh pr edit ${{ github.event.number }} --milestone "" --repo $GITHUB_REPOSITORY
       env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
\ No newline at end of file
+        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't think changes is needed - `GITHUB_TOKEN` I think works just fine https://cli.github.com/manual/gh_help_environment

Reply:
`GITHUB_TOKEN` isn't set by default https://docs.github.com/en/actions/learn-github-actions/variables (I've tripped on this)

Reply:
Oh, sorry. Yes, both `GITHUB_TOKEN` and `GH_TOKEN` work, however there is a preference for `GH_TOKEN`.

I'm happy to drop this.